File: DuDoong-Api/src/main/java/band/gosrock/api/comment/model/response/RetrieveCommentCountResponse.java
package band.gosrock.api.comment.model.response;


import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class RetrieveCommentCountResponse {

    private final Long commentCounts;

    public static RetrieveCommentCountResponse of(Long commentCounts) {
        return RetrieveCommentCountResponse.builder().commentCounts(commentCounts).build();
    }
}

File: DuDoong-Api/src/main/kotlin/band/gosrock/api/comment/model/response/RetrieveCommentCountResponse.kt
package band.gosrock.api.comment.model.response

data class RetrieveCommentCountResponse(
    val commentCounts: Long?,
) {
    companion object {
        @JvmStatic
        fun of(commentCounts: Long?): RetrieveCommentCountResponse =
            RetrieveCommentCountResponse(commentCounts = commentCounts)
    }
}
